server/project/api/events_handler.py

```diff
@@ -1,6 +1,7 @@
 from flask import Blueprint, abort
 from flask_restx import Resource, fields, marshal
-from project import db, api
+from project import db
+from project.api import api
```

Comment:
Does this serve a purpose? This is going to require a refactor of anything dependant on `api`.

Reply:
Yes this was on purpose, I refactored all imports to point to `project.api `. 
Reasons:

-  It was unnecessary to import it outside of the `create_ap`p function in `project.__init__.py`
- Because we have a folder named api, the mocking got confused when I told it to mock a function from this path `project.api.calendar_handler...` . it thought api was referring to the api imported in `project.__init__.py`

Reply:
No problem once we get this merged i will just refactor my own code.

Reply:
oh right, yes that will need to be done.